tags.md

```diff
@@ -0,0 +1,75 @@
+# Audience
+## Who is our audience (hypothesis)
+* expats/immigrants: people who moved to Porto and looking for the places outside of their homes where they can have quality time
+* curious locals: local crowd that is hungry for something new and good, usually those who speak english well, hang out with expats and traveled outside of Porto
+
+# Tags
+Tags below are required for the audience of the Porto Places to filter and find the place they want to go.
+
+## Category
```

Comment:
The service already supports the multiple modes, so that category can be either mode-specific or tag-mode itself. Probably the tag-modes will be hidden from the UI, because its obvious that all of the venues in the "food and drink" mode will have same tag

Reply:
Anyway. We will add them to each object of a place?

Reply:
yes, it's more internal type of tags, that we will not show on the UI

Reply:
ok. Will leave it like that. Let's assume that here are the tags that are going to be in data, which doesn't necessarily means it is hsown on the UI.